src/index.ts

```diff
@@ -233,7 +235,7 @@ export class XrpSettlementEngine {
       const accountJSON = await this.redis.get(`${DEFAULT_SETTLEMENT_ENGINE_PREFIX}:accounts:${accountId}`)
       if (accountJSON) {
         const account = JSON.parse(accountJSON)
-        await this.notifySettlement(account.id, drops.toString())
```

Comment:
As above, why use `Number` here?
In this case it is converted to a String again and sent in a JSON object anyway isn't it?